server/routes/aws/onboarding.py

```
@@ -304,61 +304,64 @@ def workspace_cleanup(user_id, workspace_id):
+                with db_pool.get_admin_connection() as conn:
+                    cur = conn.cursor()
+                    cur.execute(
+                        """UPDATE workspaces SET aws_discovery_summary = NULL,
+                           aws_discovery_artifact_bucket = NULL,
+                           aws_discovery_artifact_key = NULL,
+                           updated_at = CURRENT_TIMESTAMP
+                           WHERE id = %s""",
+                        (workspace_id,),
+                    )
+                    conn.commit()
+            except Exception as db_exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)